spec/bitfields_spec.rb

```diff
@@ -254,13 +323,13 @@ class ManyBitsUser < User
   end
 
   describe '#bitfield_changes' do
-    it "has no changes by defaut" do
+    it "has no changes by default" do
       User.new.bitfield_changes.should == {}
     end
 
     it "records a change when setting" do
       u = User.new(:seller => true)
-      u.changes.should == {'bits' => [0,1]}
+      u.changes.should include({ 'bits' => [0,1] })
```

Comment:
why not equal .. what else got changed ?

Reply:
With the addition of treating each bitfield as a virtual attribute, the set of changes now also includes the changed values for `seller`. 
In this case, we're changing `seller` in the initializer from false to true, which will add `'seller' => [false, true]` to the change set as well.